app.py

```diff
@@ -19,23 +20,65 @@ def exportBag():
     "sample.tsv" in its "data" sub-directory. It creates
     a new workspace in a FireCloud namespace, and uploads
     the data into it.
+    Aborts execution and returns HTTP body back to client if any 
+    request operation fails to succeed.
     """
     req_body = app.current_request.raw_body
     req_query_params = app.current_request.query_params
     req_headers = app.current_request.headers
     os.chdir('/tmp')
     with zipfile.ZipFile(io.BytesIO(req_body), 'r') as archive:
         archive.extractall()
-    payload = []
-    payload.append('/tmp/manifest/data/participant.tsv')
-    payload.append('/tmp/manifest/data/sample.tsv')
+    data = {
+        'participant': '/tmp/manifest/data/participant.tsv',
+        'sample': '/tmp/manifest/data/sample.tsv'
+    }
+    url = 'https://api.firecloud.org/api/workspaces'
     workspace = req_query_params['workspace']
     namespace = req_query_params['namespace']
-    url = 'https://api.firecloud.org/api/workspaces'
     auth = req_headers['Authorization']
-    manifest_io = ManifestIO(payload, url, workspace, namespace, auth)
-    statusCode = manifest_io.import_tsv_to_fc()
-    return Response(body=statusCode,
-                    status_code=200,
-                    headers={'Content-Type': 'application/json',
-                             'Accept': 'application/json'})
+    # Dict to collect status codes (or booleans) for successful operations.
+    statusCode = {
+        'status codes': {
+            'workspace exists': '',
```

Comment:
Wouldn't the Python convention be workspace_exists? I know it's a JSON object, so maybe Python rules don't apply, so just throwing this out there. This comment applies to the other property names as well.